gatsby-node.js

```diff
@@ -66,6 +67,14 @@ module.exports.modifyWebpackConfig = ({ config, program }) => {
 
         return current;
     });
+
+    // Setup service worker gateway
+    config.merge((current) => {
+        current.plugins.push(new ServiceWorkerWebpackPlugin({
+            entry: path.join(__dirname, 'src/shared/service-worker/sw.js'),
+        }))
+        return current;
```

Comment:
Add a new line for consistency, I guess :D

Reply:
On it!